Ansible/roles/perun-slave/tasks/install.yml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: Install Perun GPG key
-  sudo: true
-  rpm_key: key=ftp://homeproj.cesnet.cz/rpm/perunv3/RPM-GPG-KEY-perunv3 state=present
+#- name: Install Perun GPG key
```

Comment:
hm. this was my mistake actually. you can remove the Perun role for now. Do that in the UI playbook.